README.md

```diff
@@ -30,6 +30,10 @@ var Graph = require('p2p-graph')
 
 var graph = new Graph('.root')
 
+graph.on("select", function(id) {
+  console.log(id + " selected!")
+})
+
```

Comment:
Please ensure that all example code passes `standard`. So, single quotes & space after the `function` keyword.